src/com/cft/contactmerge/CompareContactParts.java

```diff
@@ -6,9 +6,14 @@ public static AnswerType doNamesMatch(String firstName1, String lastName1, Strin
         return NameMatchSelector.getFirstLastNamesMatchResultDefaultComparisons(firstName1, lastName1, firstName2, lastName2);
     }
 
-    public static AnswerType doAddressesMatch(String address1, String address2)
+    public static AnswerType doStreetAddressesMatch(Contact contact, String otherStreetAddress)
     {
-        return AddressMatchSelector.getStreetAddressMatchResultDefaultComparisons(address1, address2);
+        return contact.getAddress().isStreetAddressMatch(otherStreetAddress);
+    }
+
+    public static AnswerType doApartmentAddressesMatch(Contact contact, String otherApartmentAddress)
```

Comment:
The fact that this is returning null makes me suspect that there is no test or there is a fake test. This is fine to do on a branch but I wouldn't merge this to main if we are still planning to follow the rules we have in the Wiki.

Reply:
it's a skeleton method, it hasn't been implemented yet, and there are tests for it.